src/pages/dashboard/index.js

```diff
@@ -0,0 +1,95 @@
+import { nest } from 'd3-collection'
+import { json } from 'd3-fetch'
+
+import React, { useState, useEffect } from 'react'
+import Layout from '../../components/layout'
+import AreaChart from './_AreaChart'
+import SmallMultiplesAreaCharts from './_SmallMultiplesAreaCharts'
+import MapContainer from './_MapContainer'
+
+import { calculateTotal, parseDate } from './_util'
+
+import './dashboard.scss'
+
+function groupAndSortStateDaily(data) {
+  const grouped = nest()
+    .key(d => d.state)
+    .entries(data)
+
+  return grouped.sort((a, b) => {
+    const lastA = a.values[0]
+    const lastB = b.values[0]
+
+    const lastATotal = calculateTotal(lastA)
+    const lastBTotal = calculateTotal(lastB)
+    return lastBTotal - lastATotal
+  })
+}
+
+function transformUsDaily(data) {
+  return data
+    .map(d => {
+      const date = parseDate(d.date)
+      return [
+        {
+          date,
+          label: 'Total',
+          value: calculateTotal(d),
+        },
+        {
+          date,
+          label: 'Positive',
+          value: d.positive,
+        },
+      ]
+    })
+    .flat()
+}
+
+const DashboardPage = () => {
+  const [usDaily, setUsDaily] = useState([])
+  const [stateDaily, setStateDaily] = useState([])
+
+  useEffect(() => {
+    async function fetchData() {
+      const usDailyReq = await json('https://covidtracking.com/api/us/daily')
```

Comment:
Since Gatsby already has all the Daily, etc. data in it's database, you don't even need to do fetch request to the API - you can just query it directly.